Lesson2/Chapter4/Zad5.py

```diff
@@ -0,0 +1,21 @@
+'''
+Created on 22 lut 2018
+
+@author: agata.wiewiora
+
+For the given integer N calculate the following sum: 
+
+1^3 + 2^3 + ... + N^3
+
+'''
+
+
+N = int(input('How many numbers? '))
+sum_of_numbers = 0
+
+if type(N) == int and N > 0:
+    for nums in range(0, N+1):
```

Comment:
nazwa `nums` sugeruje, że to jakaś struktura (lista, set, krotka...), tymczasem ta zmienna przechowuje pojedynczego inta, dla czytelności zalecam zmianę nazwy np na `num`